opm/simulators/wells/StandardWellPrimaryVariables.cpp

```diff
@@ -680,7 +697,10 @@ relaxationFactorFractionsProducer(const BVectorWell& dwells) const
                 relaxation_factor *= further_relaxation_factor;
             }
         }
-        assert(relaxation_factor >= 0.0 && relaxation_factor <= 1.0);
+        if (relaxation_factor <= 0.0 && relaxation_factor > 1.0) {
```

Comment:
`<= 0.0` is intentional to see whether it actually causes problem.

Reply:
> `<= 0.0` is intentional to see whether it actually causes problem.

Okay, but maybe you meant `f <= 0.0 || f > 1.0` instead?  As written, this condition is equivalent to `if (false)`.

Reply:
changed to f < 0.0 || f > 1.0 to reflect the original behavoir.